filip/semantics/vocabulary/datatype.py

```diff
@@ -0,0 +1,136 @@
+from enum import Enum
+from typing import List, Union, Dict
+
+from . import Entity
+
+
+class DatatypeType(str, Enum):
```

Comment:
can't we reuse somehow?

Reply:
Reuuse from what? I think they are two fundamentaly different from any other existing datatype enums in the project